c7n/resources/route53.py

```diff
@@ -485,3 +486,42 @@ def process(self, resources, event=None):
             elif not logging and not state:
                 results.append(r)
         return results
+
+
+@HostedZone.filter_registry.register('enhance-hostedzone-attributes')
+class HostedZoneLogGroupAttributes(Filter):
+    """This filter type adds log group subscriptions related data to hostedzone resource output.
+    :example:
+    .. code-block:: yaml
+
+            policies:
+              - name: hostedzone-loggroup-attribute-addition
+                resource: hostedzone
+                filters:
+                  - type: query-logging-enabled
+                    state: true
+                  - type: enhance-hostedzone-attributes
+                actions:
+                  - notify
+    """
+
+    permissions = ('route53:GetHostedZone')
```

Comment:
@sgungi-intuit this is causing tests/test_iamgen.py to fail. Needs to be
```
permissions = ('route53:GetHostedZone',)
```
but yea, see my comment above. It looks like this don't belong in this PR.